### PR TITLE
JBPM-5236 - Support process migration from one runtime manager to ano…

### PR DESCRIPTION
**Pull request by mswiderski**
…ther - via kie server

### CODE REVIEW

File: kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/src/main/java/org/kie/server/remote/rest/jbpm/admin/ProcessAdminResource.java [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+
+    public ProcessAdminResource(ProcessAdminServiceBase processAdminServiceBase, KieServerRegistry context) {
+        this.processAdminServiceBase = processAdminServiceBase;
+        this.context = context;
+    }
+
+    @PUT
+    @Path(MIGRATE_PROCESS_INST_PUT_URI)
+    @Produces({MediaType.APPLICATION_XML, MediaType.APPLICATION_JSON})
+    public Response migrateProcessInstance(@javax.ws.rs.core.Context HttpHeaders headers, @PathParam("id") String containerId, @PathParam("pInstanceId") Long processInstanceId,
```
Comment (rsynek) [inline kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/src/main/java/org/kie/server/remote/rest/jbpm/admin/ProcessAdminResource.java]: This method accepts parameter "pInstanceId" for process instance ID, the next one uses "instanceId" for a list of process instance IDs. Could we name it e.g. "instanceId" and "instanceIds" to clearly communicate cardinality of these parameters?

Comment (mswiderski) [inline kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/src/main/java/org/kie/server/remote/rest/jbpm/admin/ProcessAdminResource.java]: good point, will update param names

Comment (mswiderski) [inline kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/src/main/java/org/kie/server/remote/rest/jbpm/admin/ProcessAdminResource.java]: @rsynek I aligned the name of the parameter to be same in both cases as they anyway needs to be given separately so it's better to stick to singular instead e.g.
url?pInstanceId=1& pInstanceId=2& pInstanceId=3

Comment (rsynek) [inline kie-server-parent/kie-server-remote/kie-server-rest/kie-server-rest-jbpm/src/main/java/org/kie/server/remote/rest/jbpm/admin/ProcessAdminResource.java]: @mswiderski alright, thanks.

**Comment by sutaakar** [comment]
looks good

**Pull request merged**